### PR TITLE
ESKO editorial nits

### LINKED ISSUE #42: Text nit in 3.2.2: better signal optionality of 307 response and remove word "suitable"
OLD:

> The cloud registrar replies to the voucher request with a suitable HTTP 307 response code, including the owner's local domain in the HTTP Location header.

NEW:

> In case of redirection, the cloud registrar replies to the voucher request with a HTTP 307 Temporary Redirect response code, including the owner's local domain in the HTTP Location header.

### LINKED ISSUE #43: Text nit: use of term "below"
To replace the wording "below" (which is very vague) with exact section reference.
* One reference to ""est-domain" field as defined below" -> say which section
* One reference to "Section 5 below" -> can remove the word "below".

### LINKED ISSUE #44: Section 3.3.1 unclear requirement "can validate the TLS connection"
OLD: 
> When the pledge downloads a voucher, it can validate the TLS connection

NEW: (proposed)
> After the pledge receives the voucher, it validates the TLS connection

(use of "can validate" conveys some optionality - while I think it's mandatory as defined per standard BRSKI. That aspect is already covered by referring to "per standard BRSKI operation" later in the sentence.)

### LINKED ISSUE #45: Section 3.3.1 unclear requirement "and MUST restart the process"
OLD:

> The pledge MUST process any error messages as defined in [[BRSKI](https://www.rfc-editor.org/info/rfc8995)], and MUST restart the process from it's provisioned cloud registry anchor.

NEW: (proposed)

> The pledge MUST process any error messages as defined in [[BRSKI](https://www.rfc-editor.org/info/rfc8995)], and in case of error MUST restart the process from it's provisioned cloud registry anchor.

### LINKED ISSUE #46: Harmonize terminology "owner registrar" / "Owner Registrar" / "Owner Register"
different spellings and capitalizations are used.

### PR DESCRIPTION
**Pull request by mcr**
close #43 
close #44 
close #45 
close #46 
close #42

### CODE REVIEW

File: anima-brski-cloud.md [hunk 1]
```diff
@@ -190,91 +190,91 @@ Similarly, what address space the IP address belongs to, whether it is an IPv4 o
...
 
 ### Cloud Registrar Discovery
 
-BRSKI defines how a pledge MAY contact a well-known URI of a cloud registrar if a local domain registrar cannot be discovered.
-Additionally, certain pledge types might never attempt to discover a local domain registrar and might automatically bootstrap against a cloud registrar.
+BRSKI defines how a pledge MAY contact a well-known URI of a Cloud Registrar if a local domain Registrar cannot be discovered.
+Additionally, certain pledge types might never attempt to discover a local domain Registrar and might automatically bootstrap against a Cloud Registrar.
 
-The details of the URI are manufacturer specific, with BRSKI giving the example "brski-registrar.manufacturer.example.com".
+The details of the URI are manufacturer specific, with BRSKI giving the example "brski-Registrar.manufacturer.example".
```
Comment (EskoDijk) [inline anima-brski-cloud.md:213]:
```suggestion
The details of the URI are manufacturer specific, with BRSKI giving the example "brski-registrar.manufacturer.example.com".
```

File: anima-brski-cloud.md [hunk 2]
```diff
@@ -190,91 +190,91 @@ Similarly, what address space the IP address belongs to, whether it is an IPv4 o
...
 ### Cloud Registrar Issues Voucher
 
-If the cloud registrar issues a voucher, it returns the voucher in a HTTP response with a 200 response code.
+If the Cloud Registrar issues a voucher, it returns the voucher in a HTTP response with a 200 response code.
 
-The cloud registrar MAY issue a 202 response code if it is willing to issue a voucher, but will take some time to prepare the voucher.
+The Cloud Registrar MAY issue a 202 response code if it is willing to issue a voucher, but will take some time to prepare the voucher.
 
-The voucher MUST include the "est-domain" field as defined below.
+The voucher MUST include the "est-domain" field as defined in {{RFC8366bis}}
```
Comment (EskoDijk) [inline anima-brski-cloud.md:277]:
```suggestion
The voucher MUST include the "est-domain" field as defined in {{RFC8366bis}}.
```

File: anima-brski-cloud.md [hunk 3]
```diff
@@ -295,17 +295,19 @@ If it happens that a location is repeated, then the pledge MUST fail the onboard
...
 
-The pledge MUST establish a provisional TLS connection with specified local domain registrar.
-The pledge MUST NOT use its Implicit Trust Anchor database for validating the local domain registrar identity.
-The pledge MUST send a voucher request message via the local domain registrar.
-When the pledge downloads a voucher, it can validate the TLS connection to the local domain registrar and continue with enrollment and bootstrap as per standard BRSKI operation.
+The pledge MUST establish a provisional TLS connection with specified local domain Registrar.
+The pledge MUST NOT use its Implicit Trust Anchor database for validating the local domain Registrar identity.
+The pledge MUST send a voucher request message via the local domain Registrar.
+
+After the pledge receives the voucher, it validates the TLS connection to the local domain Registrar and continue with enrollment and bootstrap as per standard BRSKI operation.
```
Comment (EskoDijk) [inline anima-brski-cloud.md:302]:
```suggestion
After the pledge receives the voucher, it validates the TLS connection to the local domain Registrar and continues with enrollment and bootstrap as per standard BRSKI operation.
```

File: anima-brski-cloud.md [hunk 4]
```diff
@@ -513,5 +515,13 @@ Whether or not this is the case, it behaves as if it was separate.
 
 It may be the case that one or more 307-Redirects have taken the Pledge from the built-in Cloud Registrar to one operated by a VAR.
 
-When the Pledge is directed to the Owner's EST {{!RFC7030}} Registrar, the Pledge validates the TLS connection with this server using the "pinned-domain-cert" attribute in the voucher.
+When the Pledge is directed to the Owner's {{!RFC7030}} Registrar, the Pledge validates the TLS connection with this server using the "pinned-domain-cert" attribute in the voucher.
```
Comment (EskoDijk) [inline anima-brski-cloud.md:518]:
```suggestion
When the Pledge is directed to the owner {{!RFC7030}} Registrar, the Pledge validates the TLS connection with this server using the "pinned-domain-cert" attribute in the voucher.
```

Comment (EskoDijk) [reply anima-brski-cloud.md:518]: updated to keep the terminology the same: "owner Registrar". (Not: Owner's Registrar)

File: anima-brski-cloud.md (hunk 1 above)
Comment (EskoDijk) [reply anima-brski-cloud.md:213]: But wait, this sentence refers to the specific BRSKI example that was given in RFC 8995.  If you don't want to make that reference, then reformulate the sentence. If referring to the 8995 example, it has to use example.com i.e. the same name.

**Review by upros — APPROVED** [review]

**Pull request merged**